modules/tensor_mechanics/doc/content/documentation/systems/BCs/CoupledPressureBC.md

```diff
@@ -0,0 +1,17 @@
+# Coupled Pressure BC
+
+## Description
+
+The boundary condition `CoupledPressureBC` applies a force computed in a variable to a mesh boundary.
+A `component` of the normal vector to the mesh surface (0, 1, or 2 corresponding to the $\hat{x}$, $\hat{y}$, and $\hat{z}$ vector components) is used to determine the direction in which to apply the traction.
+The boundary condition is always applied to the displaced mesh.
+
+The magnitude of the `Pressure` boundary condition has to be computed as an variable and coupled into this class.
+
+A set of `Pressure` boundary conditions applied to multiple variables in multiple components can be defined with the [PressureAction](/BCs/Pressure/PressureAction.md).
+
+!syntax list /BCs/Pressure objects=True actions=False subsystems=False
+
+!syntax list /BCs/Pressure objects=False actions=True subsystems=False
+
+!syntax list /BCs/Pressure objects=False actions=False subsystems=True
```

Comment:
Please use the non action include syntax for your new class here
```
!syntax parameters /BCs/CoupledPressureBC

!syntax inputs /BCs/CoupledPressureBC

!syntax children /BCs/CoupledPressureBC
```

Reply:
Thanks for catching this - that's a copy'n'paste error.